lib/ddtrace/configuration/settings.rb

```diff
@@ -21,6 +22,24 @@ class Settings
               default: -> { env_to_bool(Ext::Runtime::Metrics::ENV_ENABLED, false) },
               lazy: true
 
+      # Look for all headers by default
+      option  :propagation_extract_style,
```

Comment:
Separate thought here: would it make sense to create a distinct category of sub settings for distributed tracing? e.g. `settings.distributed_tracing.extract_style`?

If not, I think maybe at the very least we should consider adding a common prefix to categorically organize these configuration settings, e.g. `distributed_tracing_extract_style`

Reply:
Per the RFC the configuration option should be called `propagation_extract_style`

Reply:
We could still put that under a `distributed_tracing` subcategory.